server/graphql/v2/mutation/MemberInvitationMutations.js

```diff
@@ -148,6 +149,21 @@ const memberInvitationMutations = {
 
       if (args.accept) {
         await invitation.accept();
+        if ([MemberRoles.ACCOUNTANT, MemberRoles.ADMIN, MemberRoles.MEMBER].includes(invitation.role)) {
```

Comment:
I think we can put that in `MemberInvitation.accept`, IMO this is part of the business logic and we'll want to record the activity regardless of where we're accepting the invitation from.